specification/network/resource-manager/Microsoft.Network/stable/2025-03-01/networkWatcher.json

```diff
@@ -3106,6 +3110,10 @@
           "description": "Optional field to filter network traffic logs based on SrcIP, SrcPort, DstIP, DstPort, Protocol, Encryption, Direction and Action. If not specified, all network traffic will be logged.",
           "type": "string"
         },
+        "recordTypes": {
```

Comment:
>recordTypes [](http://example.com/codeflow?start=9&length=11)

can you model this as an array of enum input values ? than comma separated ones ? 

Using comma separated values will need additional validation/normalization on your side for this property